docs/source/api/python/mantid/geometry/ComponentInfo.rst

```diff
@@ -7,21 +7,29 @@ This is a python binding to the C++ class Mantid::Geometry::ComponentInfo.
 --------
 Purpose
 --------
-The purpose of the ComponentInfo object is to allow the user to access geometric information about the components which are part of beamline. A component is any physical item or group of items that is registered for the purpose of data reduction. The ComponentInfo object can be used to access information such as the total number of components in the beamline, the absolute position of a component as well as the absolute rotation of a component.
+The purpose of the ``ComponentInfo`` object is to allow the user to access geometric information about the components which are part of a beamline. A component is any physical item or group of items that is registered for the purpose of data reduction. The ``ComponentInfo`` object can be used to access information such as the total number of components in the beamline, the absolute position of a component as well as the absolute rotation of a component.
```

Comment:
``Component`` info provides tree like access to the beamline including all detectors.